docker_scripts/squash.py

```diff
@@ -518,8 +520,8 @@ def run(self):
             # without loading into Docker
             self._tar_image(self.output_path, new_image_dir)
             self.log.info("Image available at '%s'" % self.output_path)
-        else:
-            # Load image into Docker
+        if not self.output_path or self.load_output_back:
```

Comment:
For me, more readable would be:

```python
elif self.load_output_back:
```

Additionally, the `self.load_output_back` variable needs to be set to `True` by default.